spec/api-cookies-spec.coffee

```diff
@@ -0,0 +1,70 @@
+assert  = require 'assert'
+remote  = require 'remote'
+http    = require 'http'
+path    = require 'path'
+BrowserWindow = remote.require 'browser-window'
+
+describe 'cookies module', ->
+  fixtures = path.resolve __dirname, 'fixtures'
+  w = null
+  url = "http://127.0.0.1:9999"
+
+  beforeEach -> w = new BrowserWindow(show: true)
+  afterEach -> w.destroy()
+
+  it 'should get cookies', (done) ->
+    server = http.createServer (req, res) ->
+      console.log req
+      res.setHeader('Set-Cookie', ['type=dummy'])
+      res.end('finished')
+
+    server.listen 9999, '127.0.0.1', ->
```

Comment:
The server should be closed when test is done, and we should not hard code the port number since it might be taken by other processes.